js/components/portal-dashboard/student-navigator.tsx

```diff
@@ -8,35 +8,38 @@ import { getFormattedStudentName } from "../../util/student-utils";
 interface IProps {
   students: any;
   isAnonymous: boolean;
-  currentStudentIndex: any;
+  currentStudentIndex: number;
   setCurrentStudent: (studentId: string | null) => void;
   currentStudentId: string | null;
-  inResponseDetail?: boolean;
-  key?: any;
+  nameFirst: boolean;
+  filler?: boolean;
```

Comment:
I wouldn't even include this "filler" chunk as part of the component.  Just put it in the parent.  It's not clear what this prop means, and I think it's better to have the student navigator just be the bar with the controls while the chunk below it is just a div in the parent.

Reply:
![image](https://user-images.githubusercontent.com/5126913/102158474-63e46280-3e36-11eb-879f-569670562ca2.png)